spark/scripts/start-cluster-local.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#
+# A script that starts a Spark cluster locally via docker compose.
+#
+# Usage: $ ./start-cluster-local.sh
+
+set -e
+
+# Change working directory to spark/
+project_root=$(git rev-parse --show-toplevel)
+cd "${project_root}/spark"
+
+docker-compose up -V --force-recreate
```

Comment:
Why have users execute this script when it's basically just this one line? I don't like to wrap really well-known tools unless I need to hide some major complexity or I'm going to add a lot of value ahead of time. In this case, I'd have users just run `docker-compose` directly. They already know the tool, they know what it does and how to use it and how to modify the command if they need to.

Reply:
I added `start-cluster-local.sh` for symmetry given we also have the script `submit-example-local.sh`

Reply:
But, makes sense. I usually don't like to wrap single line cmds as well, but thought it'd help with documenting the steps using readable script names.